__tests__/server.test.js

```diff
@@ -196,4 +196,44 @@ describe("Testing APIs for GET /api/articles/:article_id", () => {
         });
 
 
-      })
\ No newline at end of file
+      })
+
+
+      describe("Testing APIs for POST /api/articles/:article_id/comments", () => {
+        test("POST:201 inserts a new comment for a specific article id to the db and sends the new comment back to the client", () => {
+          const newComment = {
+            username: 'butter_bridge',
+            body: 'Great article!'
+          };
+          
+          return request(app)
+          .post('/api/articles/1/comments')
+          .send(newComment)
+          .expect(201)
+          .then((response) => {
+              expect(response.body.comment.author).toBe('butter_bridge');
```

Comment:
you can do a more thorough test that this.  rather than numerous expect statements for each comment properties, why don't you test Equality to an object with a particular "shape"